app/scripts/modules/kubernetes/pipeline/stages/runJob/runJobStage.html

```diff
@@ -19,17 +19,42 @@
   </stage-config-field>
 
   <stage-config-field label="Image">
-    <docker-image-and-tag-selector
-      specify-tag-by-regex="false"
-      account="ctrl.stage.container.imageDescription.account"
-      organization="ctrl.stage.container.imageDescription.organization"
-      registry="ctrl.stage.container.imageDescription.registry"
-      repository="ctrl.stage.container.imageDescription.repository"
-      tag="ctrl.stage.container.imageDescription.tag"
-      show-registry="true"
-      on-change="ctrl.onChange"></docker-image-and-tag-selector>
+    <div class="form-group" ng-if="ctrl.hasDockerPipelineTriggers()">
+      <div>
+        <label class="sm-label">
+          Resolve from Trigger
+        </label>
+        <input type="checkbox"
```

Comment:
Well, I don't want to say that you should inline it... but you can just inline it.